components/sessions/sources/new-session-started/new-session-started.mjs

```diff
@@ -0,0 +1,61 @@
+import { DEFAULT_POLLING_SOURCE_TIMER_INTERVAL } from "@pipedream/platform";
+import sessions from "../../sessions.app.mjs";
+
+export default {
+  key: "sessions-new-session-started",
+  name: "New Session Started",
+  description: "Emit new an event when a session starts. [See the documentation](https://api.app.sessions.us/api-docs/#/default/get_api_sessions)",
+  version: "0.0.1",
+  type: "source",
+  dedupe: "unique",
+  props: {
+    sessions,
+    db: "$.service.db",
+    timer: {
+      type: "$.interface.timer",
+      default: {
+        intervalSeconds: DEFAULT_POLLING_SOURCE_TIMER_INTERVAL,
+      },
+    },
+  },
+  methods: {
```

Comment:
Most of the behavior and code is the same between these two sources, I suggest moving them to a common file and each source modifying/defining only what is relevant